src/heartbeat/listener.js

```diff
@@ -13,6 +13,8 @@ const debug = require('debug')('refocus-collector:heartbeat');
 const logger = require('winston');
 const utils = require('./utils');
 const configModule = require('../config/config');
+const bufferedQueue = require('buffered-queue');
+const sampleQueue;
```

Comment:
I thought we decided that we will be creating a buffered queue in its own file. Something like.. bufferedQueue/sampleBufferedQueue.js

Reply:
Ohh I thought we decided to add over hearthBeatListener but yeah I can do that as well.  But flow will be the same right ? Just to copy and paste. @shriramshankar